benchmarks/src/jmh/java/io/grpc/benchmarks/CancellableInterceptor.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.benchmarks;
```

Comment:
it would be cool if the package matched the class being benchmarked, like junit tests 


perhaps `core/src/jmh/java/io/grpc`  and a matching package?

Reply:
These benchmarks are closer to the scale of integration tests, so at the moment this location seems about as good as anywhere else. We could possibly make TransportBenchmark abstract though and then have each transport extend it, like we do with AbstractTransportTest.

But that's really unrelated to the changes, which is trying to do something with the `benchmarks/src/jmh/java/io/grpc/benchmarks/netty/` benchmarks (all of which have no reason to be Netty-specific).